generic/tests/netperf.py

```diff
@@ -392,29 +303,16 @@ def start_test(server, server_ctl, host, clients, resultsdir, test_duration=60,
     :param sizes_rr: request/response sizes (TCP_RR, UDP_RR)
     :param sizes: send size (TCP_STREAM, UDP_STREAM)
     :param protocols: test type
-    :param ver_cmd: command to check kvm version
     :param netserver_port: netserver listen port
     :param params: Dictionary with the test parameters.
     :param server_cyg: shell session for cygwin in windows guest
     """
     if params is None:
         params = {}
 
-    guest_ver_cmd = params.get("guest_ver_cmd", "uname -r")
     fd = open("%s/netperf-result.%s.RHS" % (resultsdir, time.time()), "w")
-
-    test.write_test_keyval({'kvm-userspace-ver': ssh_cmd(
-                            host, ver_cmd).strip()})
-    test.write_test_keyval({'guest-kernel-ver': ssh_cmd(
-                            server_ctl, guest_ver_cmd).strip()})
-    test.write_test_keyval({'session-length': test_duration})
-
-    fd.write('### kvm-userspace-ver : %s\n' % ssh_cmd(host,
-                                                      ver_cmd).strip())
-    fd.write('### guest-kernel-ver : %s\n' % ssh_cmd(server_ctl,
-                                                     guest_ver_cmd).strip())
-    fd.write('### kvm_version : %s\n' % os.uname()[2])
-    fd.write('### session-length : %s\n' % test_duration)
+    netperf_base.env_version(test, params, host, server_ctl,
```

Comment:
Recommend to name it `record_env_version` or `append_env_version`

Reply:
I have updated.